src/Sylius/Behat/Page/Admin/Crud/IndexPage.php

```diff
@@ -76,6 +76,14 @@ public function isResourceOnPage(array $parameters)
     }
 
     /**
+     * @return int
+     */
+    public function countTableRecords()
```

Comment:
I meant that on the interface, we of course can use table accessor in our index page implementation, but the interface should not assume that it is a table.